test/authz-filler/main.go

```diff
@@ -0,0 +1,92 @@
+// A quick way to fill up a database with a large number of authz objects, in
+// order to manually test the performance of the expired-authz-purger.
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"time"
+
+	"github.com/letsencrypt/boulder/cmd"
+	"github.com/letsencrypt/boulder/core"
+	"github.com/letsencrypt/boulder/sa"
+)
+
+type fillerConfig struct {
+	Filler struct {
+		cmd.DBConfig
+		Parallelism uint
+	}
+}
+
+type model struct {
+	core.Authorization
+
+	LockCol int
+}
+
+func main() {
+	configPath := flag.String("config", "config.json", "Path to Boulder configuration file")
+	flag.Parse()
+
+	configJSON, err := ioutil.ReadFile(*configPath)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to read config file '%s': %s\n", *configPath, err)
+		os.Exit(1)
+	}
+
+	var config fillerConfig
+	err = json.Unmarshal(configJSON, &config)
+	cmd.FailOnError(err, "Failed to parse config")
+
+	// Configure DB
+	dbURL, err := config.Filler.DBConfig.URL()
+	cmd.FailOnError(err, "Couldn't load DB URL")
+	dbMap, err := sa.NewDbMap(dbURL, 1000)
+	cmd.FailOnError(err, "Could not connect to database")
+
+	dbMap.AddTableWithName(model{}, "pendingAuthorizations").SetKeys(false, "ID")
```

Comment:
The table mapping for `pendingAuthorizations` from `sa/database.go:initTables` also calls `SetVersionCol("LockCol")` - is that useful here? Likely not since there isn't the potential for concurrent updates but I wanted to verify.

Reply:
Correct, it's not needed here since we're not doing any updates.